tools/setup-unit

```diff
@@ -421,7 +421,12 @@ unit_ctl_edit()
     fi;
     local req_path="$1";
 
-    local tmp="$(mktemp)";
+    echo "$req_path" \
+    | sed 's%^/js_modules\>.*%.js%' \
```

Comment:
I think this needs a trailing slash, because `/js_modules` is a list of objects. We only want to edit as raw JS when we specify a module by name.

Also consider that this will be a blob of text with escaped newlines and quotation marks - so not very useful in a text editor.

Reply:
Thanks!  I think this should do:

```diff
diff --git a/tools/setup-unit b/tools/setup-unit
index 0eaf0110..a0dafe10 100755
--- a/tools/setup-unit
+++ b/tools/setup-unit
@@ -422,7 +422,7 @@ unit_ctl_edit()
     local req_path="$1";
 
     echo "$req_path" \
-    | sed 's%^/js_modules\>.*%.js%' \
+    | sed 's%^/js_modules/.*%.js%' \
     | sed 's%^/config\>.*%.json%' \
     | sed 's%^/.*%.txt%' \
     | xargs mktemp --suffix \
```

I'll push in a moment.